contrib/babelfishpg_tsql/sql/sys_procedures.sql

```diff
@@ -164,3 +164,19 @@ $$ LANGUAGE plpgsql;
 GRANT EXECUTE ON PROCEDURE sys.sp_babelfish_configure(
 	IN varchar(128), IN varchar(128), IN varchar(128)
 ) TO PUBLIC;
+
+CREATE OR REPLACE PROCEDURE sys.sp_addrole(IN sys.nvarchar(256))
+       AS 'babelfishpg_tsql', 'sp_addrole' LANGUAGE C;
```

Comment:
Nit: indentation unnecessary here

Reply:
Fixed it in next revision by removing unnecessary indentations in all the created procedures.

Reply:
Fixed in next revision